core/templates/dev/head/exploration_editor/exploration_editor.html

```diff
@@ -257,11 +257,11 @@
         </p>
         <form name="objectiveForm">
           <p ng-if="requireObjectiveToBeSpecified"
-             ng-class="{'has-error': explorationObjectiveService.displayed.length <= 15}">
+             ng-class="{'has-error': explorationObjectiveService.displayed.length < 15}">
             Add a goal: what does the exploration help people to do?
-            <input name="objectiveInput" type="text" class="form-control" required ng-model="explorationObjectiveService.displayed"
+            <input name="objectiveInput" type="text" class="form-control" ng-model="explorationObjectiveService.displayed"
```

Comment:
NB: removed 'required' because that causes the value of explorationObjectiveService.displayed to end up being undefined instead of ''; this also leads to a backend error.